addon/templates/components/bs4/bs-accordion/title.hbs

```diff
@@ -0,0 +1,8 @@
+{{! template-lint-disable nested-interactive }}
+<div role="tab" class="card-header {{if collapsed "collapsed" "expanded"}}">
```

Comment:
Btw, you cannot use the `tagName: ''` approach here, as that would make the `click` handler on the component not work anymore (cannot have an event listener on a tagless component obviously).